roles/kubernetes/node/defaults/main.yml

```diff
@@ -24,6 +24,9 @@ kubelet_enforce_node_allocatable: "\"\""
 # Note that PID namespace sharing requires docker >= 1.13.1.
 kubelet_disable_shared_pid: true
 
+### fail with swap on (default true)
+kubelet_fail_with_swap: true
```

Comment:
Sorry to nitpick, but this var should be closer to the actual option name. Can you use `kubelet_fail_swap_on`?